code/modules/cm_marines/smartgun_mount.dm

```diff
@@ -53,8 +53,11 @@
 	icon_state = "M56D_gun_e"
 	var/rounds = 0 // How many rounds are in the weapon. This is useful if we break down our guns.
 
-	New()
-		update_icon()
+	
+/obj/item/device/m56d_gun/Initialize()
```

Comment:
atomise

Reply:
It literally won't work with this click PR, I am not leaving things broken just so there's a few less files changed. That would be a terrible idea.